src/Nethermind/Nethermind.Synchronization.Test/ParallelSync/MultiSyncModeSelectorTests.cs

```diff
@@ -1029,6 +1047,16 @@ public void Can_switch_to_a_better_branch_while_full_synced()
                 .ThenInAnyFastSyncConfiguration()
                 .TheSyncModeShouldBe(SyncMode.Full);
         }
+        
+        [Test]
+        public void Should_not_switch_to_same_branch_while_full_synced_but_wrong_total_difficulty()
+        {
+            Scenario.GoesLikeThis()
```

Comment:
if I read this test description then it does not make sense -> it would suggest that by learning about bad peers we are stopping th sync process even if there are valid peers

Reply:
ok, I think you want to say: OnlyPeersWithWrongDifficultyAreKnown

Reply:
changing test name to: Should_not_sync_when_synced_and_peer_reports_wrong_higher_total_difficulty
but other peers can be know too, so don't like OnlyPeersWithWrongDifficultyAreKnown name